packages/typespec-python/src/emitter.ts

```diff
@@ -933,6 +938,10 @@ function mapCadlType(context: SdkContext, type: Type): any {
                 valueType: emitSimpleType(context, sdkType.valueType),
             };
         case "Model":
+            // for "model MyModel is Record<> {}", we shall treat it as model instead of dict
```

Comment:
Will we distinguish `extends Record<unknown>` with `extends Record<string>`?

Reply:
No, since DPG models inherit from `MutableMapping[str, typing.Any]`https://github.com/Azure/autorest.python/blob/b021bef1ecba7cc9461f7b274ed185c46d0c9431/packages/typespec-python/test/generated/azure-core-basic/_specs_/azure/core/basic/_model_base.py#L275